client/assets/components/document/document_mail/document_mail.html

```diff
@@ -1,6 +1,6 @@
 <div class="detail vertical grid-content">
   <!-- show head field linked with a url. -->
-  <h3 class="head-field" ng-if="doc.subject"><a href="#"
+  <h3 class="head-field" ng-if="doc.subject"><a href="http://asfmail.lucidworks.io/"
```

Comment:
@xiaoyehhuang If you look on the documents that are displayed, you should see that there is either an ID or a URL field on the document.  The id would be to take the user directly to the resulting document.  Say, for instance, the ID is http://asfmail.lucidworks.io/mail_files/curator-user/201605.mbox/raw/%3CCANtJApz-c28BbFkGzFf6PxieYk_V_-==xJcc_5nO47h_XfPpwQ@mail.gmail.com%3E, we would take them to http://asfmail.lucidworks.io/mail_files/curator-user/201605.mbox/%3CCANtJApz-c28BbFkGzFf6PxieYk_V_-==xJcc_5nO47h_XfPpwQ@mail.gmail.com%3E (notice it's not the RAW version.

Also, note that the Website docs have the same problem.